plasmapy/physics/tests/test_distribution.py

```diff
@@ -104,16 +104,6 @@ def test_std(self):
         T_distri = (std**2 / k_B * m_e).to(u.K)
         assert np.isclose(T_distri.value, self.T_e.value)
 
-    def test_valErr(self):
```

Comment:
Why remove these tests?

Reply:
The exception changed to the one from inside thermal speed. Since we're calling thermal speed anyway and that exception is checked in its tests, I figured it's not really necessary to duplicate that check since it isn't really adding anything. I could restore them with the exception changed if necessary.

I guess I was just feeling minimalistic.

Reply:
fair enough